sdk/src/main/java/com/memfault/cloud/sdk/MemfaultOtaPackage.kt

```diff
@@ -2,12 +2,18 @@ package com.memfault.cloud.sdk
 
 /**
  * An OTA package, returned by Memfault's server for a specific [MemfaultDeviceInfo].
+ *
+ * @param size the size of the artifact at the [location]
+ * @param isDelta whether this package corresponds to a Delta or a Full Release.
  */
 data class MemfaultOtaPackage(
     val location: String,
     val releaseNotes: String,
     val appVersion: String,
     val md5: String,
     val extraInfo: Map<String, String>,
```

Comment:
I surfaced both for the sake of backwards compatibility -- maybe someone is using the (artifact's) extra_info and we don't want to break it. But then again, maybe that's fine?